src/clj/rems/db/blacklist.clj

```diff
@@ -40,6 +42,30 @@
 (defn add-event! [event]
   (db/add-blacklist-event! {:eventdata (event->json event)}))
 
+(defn- command->event [command user-id]
```

Comment:
when I wrote this stuff I thought rems.db.blacklist would only expose an event-level api, and the command could stay in the rems.api.blacklist ns

Reply:
Ok, I can move them back to the other ns.